test/__snapshots__/Storyshoots.spec.js.snap

```diff
@@ -10,19 +10,19 @@ exports[`Storyshots Components.Card Base 1`] = `
       style={Object {}}
```

Comment:
Is this file named incorrectly?  Storyshoots.spec.js.snap.

Reply:
this file is autogenerated